### ISSUE TITLE
Cluster Labels

### ISSUE THREAD

**Issue by RicherMans** [sangho-vision/acav100m#5]
Hey guys,
first of all thanks for the excellent work! The amount of data you provide is really on another level.
I recently downloaded most of the data myself, but was curious about one thing: Can you provide the original n-centroid labels i.e., for the audio/visual clusters?
I only found the download .csv files with the youtube-ids but not the corresponding cluster ids for most samples. 
Did I miss something?
Kind regards,
Heinrich

**Comment by JiwanChung** [comment]
Hi!
First of all, we are sorry that we no longer have access to the intermediate cluster labels.

With that said, I will leave some quick reminders to check we are on the same page.
1. The cluster labels are not necessary for the downstream usage of our dataset.
2. While we show in the paper that some clusters align with human semantics, we do not recommend using the cluster labels for the downstream tasks.

If necessary, you can use our feature extraction -> clustering code on the downloaded videos to get cluster labels. However, these new labels would not be the same as the original labels since the original clusters are computed on the uncurated set of videos.

Just in case you are also looking for ways to videos given the csv files, I suggest using the following library.
https://github.com/iejMac/video2dataset/blob/main/dataset_examples/ACAV.md

**Comment by RicherMans** [comment]
Hey @JiwanChung ,
thanks for the quick answer !

> The cluster labels are not necessary for the downstream usage of our dataset.

Yeah, we are on the same page, I was aware that these labels are only an "intermediate" output of your method.

> While we show in the paper that some clusters align with human semantics, we do not recommend using the cluster labels for the downstream tasks.

Yeah, I agree that they might not align with human semantics.
My initial thought here was simply to check if a "self-supervised" version of my pretrained models using ACAV100M are better than models trained on the `cluster_id`.
Since my compute is far more limited than yours, I can't imagine possibly computing embeddings, storing them and then running a clustering algorithm :(.

> If necessary, you can use our feature extraction -> clustering code on the downloaded videos to get cluster labels. However, these new labels would not be the same as the original labels since the original clusters are computed on the uncurated set of videos.

Alright, thanks for the info!

I close the issue, thanks again for clarification!
